src/app/api/contributors/team/fetchTeamContributors.ts

```diff
@@ -27,3 +27,38 @@ export async function fetchTeamContributors(
   });
   return teamContributorDto;
 }
+
+export async function fetchTeamScoreSum(teamId: string): Promise<number> {
```

Comment:
let's rename the previous function on line 8 from
fetchTeamContributors
to
fetchAllUsersTeamsContributors

or something like that,
it's quite confusing to know what is the difference between these two functions